neqo-crypto/build.rs

```diff
@@ -177,10 +175,46 @@ fn static_link() {
         } else {
             "plds4"
         },
-        "smime",
         "softokn_static",
         "ssl",
     ];
+    // macOS always dynamically links against the system sqlite library.
+    // See https://github.com/nss-dev/nss/blob/a8c22d8fc0458db3e261acc5e19b436ab573a961/coreconf/Darwin.mk#L130-L135
+    if env::consts::OS == "macos" {
```

Comment:
When cross-compiling, `env::consts::OS` might not be the target OS.

I guess in this particular case it doesn't matter, but more generally, I assume using `env::var("CARGO_CFG_TARGET_OS")` would be more accurate?